src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -294,28 +342,12 @@ def _get_endpoint_and_token_status(cmd, login_server, ignore_errors):
     print_pass("Fetch refresh token for registry '{}'".format(login_server))
     print_pass("Fetch access token for registry '{}'".format(login_server))
 
-
-def _check_registry_health(cmd, registry_name, ignore_errors):
+def _check_registry_health(cmd,
+                           registry_name,
+                           registry,
```

Comment:
Why do you need registry and registry name?

Reply:
'Registry' is returned from the get_registry_by_name method, and the registry properties are then accessed by the subfunctions. 

Resgistry_name is just the name of the registry